eogrow/core/config.py

```diff
@@ -103,10 +106,6 @@ def interpret_config_from_dict(config: CrudeConfig, external_variables: dict[str
     config_with_variables = _recursive_apply_to_strings(
         config, lambda config_str: _resolve_variables(config_str, variable_mapping)
     )
-    if not isinstance(config_with_variables, dict):
-        raise ValueError(
-            f"Interpretation resulted in object of type {type(config_with_variables)} but a dictionary was expected."
-        )
```

Comment:
This was removed because it's now handled elsewhere, right (IIRC)?

Reply:
it's handled 10 lines earlier, at the start of the function